src/directives/popover.js

```diff
@@ -0,0 +1,137 @@
+goog.provide('ngeo.popoverDirective');
+goog.provide('ngeo.popoverAnchorDirective');
+goog.provide('ngeo.popoverContentDirective');
+goog.provide('ngeo.PopoverController');
+
+goog.require('ngeo');
+
+
+/**
+ * Provides a directive used to display a Bootstrap popover.
+ *
+ * @example <caption> Basic usage of nego popover </caption>
+ *<div ngeo-popover>
+ *  <a ngeo-popover-anchor class="btn btn-info">anchor 1</a>
+ *  <div ngeo-popover-content="">
+ *    <ul>
+ *      <li>action 1:
+ *        <input type="range"/>
+ *      </li>
+ *    </ul>
+ *  </div>
+ *</div>
+ * @ngdoc directive
+ * @ngInject
+ * @ngname ngeoPopover
+ * @return {angular.Directive} The Directive Definition Object.
+ */
+ngeo.popoverDirective = function() {
+  return {
+    restrict: 'A',
+    scope: {},
+    controller: 'NgeoPopoverController',
+    link: function(scope, elem, attrs, controller) {
+      function onClick(clickEvent) {
+
+        if (controller.anchorElm[0] !== clickEvent.target && controller.shown) {
+          controller.shown = false;
+          controller.anchorElm.popover('hide');
+        }
+
+      }
+
+      angular.element('body').on('click', onClick);
+    }
+  }
+};
+
+/**
+ * @ngdoc directive
+ * @ngInject
+ * @ngname ngeoPopoverAnchor
+ * @return {angular.Directive} The Directive Definition Object
+ */
+ngeo.popoverAnchorDirective = function() {
+  return {
+    restrict: 'A',
+    require: ['^^ngeoPopover'],
+    scope: {},
+    link: function(scope, elem, attrs, controllers) {
+      var ngeoPopoverCtrl = controllers[0];
+      ngeoPopoverCtrl.anchorElm = elem;
+      elem.on('inserted.bs.popover', function(event) {
+        ngeoPopoverCtrl.shown = true;
+        angular.element('.popover').on('click', function(e) {
+          e.stopPropagation();
+        })
+      });
+
+      elem.on('hidden.bs.popover', function(event) {
+        /**
+         * @type {{inState : Object}}
+         */
+        var popover = elem.data('bs.popover');
+        popover['inState'].click = false;
+      });
+    }
+  }
+};
+
+/**
+ * @ngdoc directive
+ * @ngInject
+ * @ngname ngeoPopoverContent
+ * @return {angular.Directive} The Directive Definition Object
+ */
+ngeo.popoverContentDirective = function() {
+  return {
+    restrict: 'A',
+    transclude: true,
+    require: ['^^ngeoPopover'],
+    scope: {},
+    link: function(scope, elem, attrs, controllers, transclude) {
+      var ngeoPopoverCtrl = controllers[0];
+      ngeoPopoverCtrl.bodyElm = transclude();
```

Comment:
Can you try without transclude and just set `ngeoPopoverCtrl.bodyElm` from element first child.
Translude is actually used for something specfic that is not your usecase.
I wonder if the transclude function doesn't ensure the content is compiled though.
To test.